src/angular-sql-query.js

```diff
@@ -779,14 +808,27 @@
       return resources;
     }
 
+    function path(p, value) {
```

Comment:
Why wouldn't you add ramda or any fp library instead of rewriting few functions?

Reply:
It's because this lib is still installed with bower and I want to avoid to add dependencies :/

Reply:
and an other technical ticket for us!